### PR TITLE
Modernize navbar

### LINKED ISSUE #14: Modernize navbar
Right now the navbar is hugely dependent on javascript in mobile view. Maybe refactor it to have less javascript dependency.

### PR DESCRIPTION
**Pull request by triszt4n**
* Actually functional and pretty navbar
* Weeded out not-so-useful scss classes (basically those scss files from old repo can be improved and are redundant)
* Applied gatbsy's recommended changes (added gatsby-image-plugin and added keys to listed items on index page)
* Added translations for the navbar (owner can change it later) as an example for later implementations (although i18next suggests on their website the use nested keys, in a github issue they realized they didn't make it work)

### CODE REVIEW

**Review by martonoravecz — DISMISSED** [review]
I do not see issues, but a second opinion by @SepsiLaszlo  or @poiuztrewq1 would be appreciated due to limited frontend knowledge on my part.

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 import React from 'react'
+import SEO from '../components/SEO'
+import Layout from '../layout/Layout'
 
 const ContactPage: React.FC<PageProps> = () => {
-  return <div>asd</div>
+  return (
+    <Layout>
+      <SEO />
+      <div>asd</div>
```
Comment (bbucsy) [inline src/pages/contact.tsx]: Is `asd` intentional? I think a "TODO" would be less cringe if we leave it in by mistake and some IDE can highlight left in `//TODO` comments

**Review by bbucsy — DISMISSED** [review]
Also small bug, might not be important: 
On (_Firefox/Ubuntu_) you can break the navbar by using the side scrollbar instead of the mouse wheel

Otherwise approved :ok_hand:

File: src/pages/contact.tsx (hunk 1 above)
Comment (triszt4n) [reply src/pages/contact.tsx]: Applied it in new commit.

Also, some browsers do not emit perfectly the scrollTop property of the window element (optimization I guess), so yeah, one can f*ck up the navbar by dragging it crazy.

**Review by bbucsy — APPROVED** [review]
:metal:

**Pull request merged**